### DISCUSSION TITLE
Discussion board discussion

### DISCUSSION THREAD

**Discussion by CalvinRodo** [cds-snc/covid-alert-documentation#55] [General]
This comment was originally posted by LeCanardQuoi from the following issue #416  which is now a locked thread for informational purposes only. 

>Process/meta discussion can also be a source of distraction with limited benefits in the middle of the 2nd wave of a pandemic preventing from delivering a highly used app that will make a real difference for managing the pandemic. There is not a whole of time for this to happen and properly baseline a process that should have been done by now. A proper baseline will be better established once the second wave is over in preparation to a third wave, or even a new type of pandemic. It must be pointed out here that the existing process (whatever it is) has failed to timely deliver the requirements for BC and Alberta. In addition, the support of the app by provinces signing up and their regional health authorities has been lukewarm at best thus far. There will be serious consequences to these shortcomings during the second wave.

>If developpers are getting distracted by enhancement requests, they should avoid them until proceeding to pull requests stage, after due diligence review on the github with principal stakeholders actively participating: HC, canadian advisory council, CDS product manager&release manager, provincial health authority rep (when applicable), core users, etc. 

>The most useful thing we could have at this point imho would be a rough plan committed to by the appropriate stakeholders that describes how the github server is expected to be used over the next 3 months to better manage and deliver new requirements (from enhancement requests to pull requests and actual maintenance of line releases ). Only then will it be realistic to expect the github to make a real difference for delivering a highly used app. As an "app core user", I do not own the said rough plan but can only contribute review insights. Other said stakeholders actually participating onto this thread would be an indicator of the likelyhood of this to happen. Someone has to own this rough plan and be tasked to bring it to fruition. Creating a github issue is likely most insufficient. These other stakeholders would also have to be tasked to bring it to fruition, if this is taken seriously. A CDS Director and an HC Director have to get involved to make this happen. It will not happen otherwise. A high level will to deliver a highly used app for better managing the pandemic remains most unclear.

**Comment by LeCanardQuoi** [comment]
Following enhancement requests may have been hindered by existing requirement management process (perhaps in combo with insufficient development resources&budget and lack of high level visibility) on the app component side 
#1264, #1304, #1305 (and #391 server side), #1328, #1329, #1332, #1345, #1347 and #1380.
Entering new ERs is now hindered as a result of uncertainties managing existing queue of ERs.

**Comment by LeCanardQuoi** [comment]
I hereby propose that the Covid Alert Advisory council establish a working group to steer requirements, design, field trials and deployment associated with integrating QR scanning functionality within the existing #COVIDAlert system

**Reply by CalvinRodo** [reply]
We appreciate the proposal however this repo is for technical work on Covid Alert Server, if you have proposals for how the Covid Alert Advisory council should work you will need to contact your local MP.

**Comment by LeCanardQuoi** [comment]
It would appear that some of the process stuff is in the process of being documented under https://github.com/orgs/cds-snc/projects/9#card-53588891
Still unclear how new requirements are reviewed, who actually reviews, if and how the advisory council (workgroup), hc and core users are involved. The QA process is also unclear as new reqs (e.g. QR scan)  have been appearing in the repository without minimum content specified earlier on. Are unaccessible "Notes" now used to shield away meaningful info from the Open Source repository?  In addition, many issues are opaque in the menu lists for open and closed entries. The QA lead should really participate too on the github server, and certainly on this particular thread.

**Reply by CalvinRodo** [reply]
This is just a reminder that this project is only for technical requirements and issue tracking for the back end infrastructure of Covid Alert, just as https://github.com/cds-snc/covid-alert-app is only being used to track work for the Mobile app and https://github.com/cds-snc/covid-alert-portal is being used to track work for the Healthcare integration Portal. 

Prioritization and planning of high level requirements for the Covid Alert System do not occur here.

This project is also only used by staff at the Canadian Digital Service (CDS) and is not being monitored by Health Canada, or the App Advisory Council. 

If you have concerns around how this project is being run I recommend you contact your local member of parliament.

**Reply by LeCanardQuoi** [reply]
Scope limited to app server development was most unclear when cds-snc/covid-alert-server#415 and #416 were first created following the icebreaker call in which you did not incidentally participate. The Open Source github repository provides a public record/trace of issues raised, whether related to the management of new requirements or QA process. Other channels are used to escalate as needs be.

**Reply by CalvinRodo** [reply]
I don't mean to be dismissive of your comments but at the moment there is no avenue for escalation through this Github Repo.

As long as you are abiding by the [repositories code of conduct](https://github.com/cds-snc/covid-alert-server/blob/master/CODE_OF_CONDUCT.md) you are free to interact with this repository. 

I would ask though if you could keep the interactions with this repository scoped to the technical work associated with this server. 

You can continue to post whatever you like to this repository but I would say that if you are looking to impact how the Covid Alert project is managed or how the product actually works I think that your best approach would be to contact your local member of provincial parliament.  

If you are interested in providing enhancements to how the Covid Alert Server works we always welcome and review all pull requests.

**Comment by LeCanardQuoi** [comment]
As an other example of problematic current release process, issue #1173 raised way back in October would have allowed reaching a whole lot more of legacy Android mobile users (and should have been released alongside the older ios support upgrade claiming huge support of older phones). Fix/test was straightforward and stalled until pull request #1426 finally build it now. Messaging required to properly release that should go far beyond a maintenance of line (MOL) release and could be part of this larger app relaunch proposed earlier on. Note that these old devices could provide a relevant "mobile phone loaner" solution for universal access solution (over and above providing for legacy users not fortunate enough to upgrade) in support of higher app adoption rates. A disclaimer on security on the older Android API should be released along with proper release notes. Core user associated with #1173 (i.e. Steve M.) should at minimum be notified properly of final release scheme. Core users are precious and should be better recognized ;-) A core users group active on the github could certainly be established by the advisory council and assist on scrubbing development issues and release planning, a gaping hole in current processes.

**Reply by CalvinRodo** [reply]
If you have concerns around the governance for Covid Alert I recommend that you contact your local MP.

**Comment by obrien-j** [comment]
Locking this thread as it's gone off topic a few times, and would likely be best redirected into a few more clearly defined topics.
